### DISCUSSION TITLE
Код ошибки: 7 Сообщение: No required scope

### DISCUSSION THREAD

**Discussion by virtua10ne** [imercury13/ya360#48] [Q&A] [answered]
Привет.

В ответ на whois или user show получаю "Код ошибки: 7 Сообщение: No required scope"
Пользователь авторизуется по SSO и присутствует в списке пользователей Я360.

```
> ya360 whois romangr1
Код ошибки: 7 Сообщение: No required scope
> ya360 whois grigoriy.romanenkov
Код ошибки: 7 Сообщение: No required scope
> ya360 user show grigoriy.romanenkov@tablogix.ru
Код ошибки: 7 Сообщение: No required scope
```

Такое ощущение, что что-то недонастроено.

**Comment by imercury13** [comment, marked as answer]
Привет.
Да все не дойдут руки переписать документацию :-( Вы правы, именно не хватает прав, которые указывались при регистрации приложения. Необходимо указывать весь перечень скопов:

directory:write_users
directory:write_departments
directory:write_groups
ya360_admin:mail_write_user_settings
ya360_admin:mail_write_shared_mailbox_inventory
directory:write_domains
directory:manage_dns
ya360_security:domain_passwords_write
ya360_security:domain_sessions_write
ya360_security:domain_2fa_write
ya360_admin:mail_write_antispam_settings
ya360_admin:mail_write_routing_rules
